hydra-cluster/src/CardanoCluster.hs

```diff
@@ -136,88 +117,25 @@ withCluster tr cfg@ClusterConfig{parentStateDirectory} action = do
     makeNodesConfig parentStateDirectory systemStart
       <$> randomUnusedTCPPorts 3
 
-  withBFTNode tr cfgA $ \nodeA -> do
-    withBFTNode tr cfgB $ \nodeB -> do
-      withBFTNode tr cfgC $ \nodeC -> do
+  withBFTNode (nodeTracer cfgA) cfgA $ \nodeA -> do
```

Comment:
Suggestion: only pass `cfgA` and retrieve tracer from it inside `withBFTNode`?

Reply:
Hmm, ok, it does not work the way I thought it worked :) Maybe this is more confusing than the previous version 🤔